linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -638,6 +642,7 @@ static int eth_vlan_application_register(struct ipcp_instance_data * data,
                 return -1;
         }
 
+        data->blocking = blocking;
```

Comment:
This seems mistaken. The "blocking" attribute should be associated to a registered application, not to a whole IPCP instance. It's true that shim-eth-vlan can allow just an application to be registered... but it's really confusing, we should add a comment at least.